assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -321,7 +273,7 @@ table.wc-block-cart-items {
 					display: inline-block;
 				}
 
-				.wc-block-cart-item__sale-badge {
+				.wc-block-sale-badge {
```

Comment:
Should some of these styles be in the components directly? Probably not a big deal, there's a little mixing of concerns here - the cart is overriding/tweaking component styles.

Reply:
I don't think so, I left them here in purpose. Even though they refer to the `ProductSaleBadge` component, they are only relevant under the _Cart_ block, so I think they belong to the _Cart_ CSS.